tests/TestCase/ORM/Association/HasOneTest.php

```diff
@@ -173,8 +173,7 @@ public function testAttachToMultiPrimaryKey()
                 'table' => 'profiles',
             ],
         ]);
-        $query->expects($this->never())->method('select');
-        $association->attachTo($query, ['includeFields' => false]);
```

Comment:
This test is unrelated to `includeFields`. The test passes with our without this setting and avoiding the `select()` call is irrelevant. Don't know if this was copied from somewhere else.